hw/top_earlgrey/top_earlgrey_verilator.cc

```diff
@@ -29,6 +29,7 @@ int main(int argc, char **argv) {
       "gen_flash_banks[0].u_flash.gen_generic.u_impl_generic.u_mem.gen_generic."
       "u_impl_generic");
   simctrl.RegisterExtension(&memutil);
+  simctrl.SetInitialResetDelay(100);
```

Comment:
added a long delay here to cover the initial reset release time.
This number will likely change in the future when the reset manager uses a slower clock.  Maybe it's best long term to calculate this out of the existing clocks in the system.

Reply:
@vogelpi 
this still feels like a weird work around to me... not sure if you guys have a different suggestion.
Since verilator doesn't seem to support async resets or X's, it feels like i'm relying on a default state of 0 to just work...

Reply:
I think you are referring to #1884 here (I updated also the issue).

Ibex needs a negative edge on the reset to reset the privilege register (actually any register). Due to
- Verilator not supporting X and resolving this to 0 in this case,
- the reset synchronizer flops being to 0, and
- the default initial reset delay being just 2 cycles,
the synchronizer flops never output anything else than 0. I.e. the core does not get the required negative edge on the reset (the sync flop would output the first one after 2 cycles, but then the reset becomes active again).

I think you are doing the right thing.